src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  static alive = [];
+  static deleteIndex = Animal.alive.findIndex((animal) => animal.health <= 0);
```

Comment:
There is an issue with the `deleteIndex` static property. It calculates the index only once when the class is first loaded, which is not the intended behavior. Instead, the index should be recalculated each time an animal's health needs to be checked, which can be done inside a method.